src/Sulu/Bundle/AdminBundle/Resources/js/containers/Form/fields/SingleSelection.js

```diff
@@ -132,6 +135,10 @@ export default class SingleSelection extends React.Component<Props>
             throw new Error('The "form_options_to_list_options" option has to be an array if defined!');
         }
 
+        if (requestParameters && !Array.isArray(requestParameters)) {
```

Comment:
this can never be `null`/`undefined` as you set the default value to `[]` in the destructing assignment above, therefore you dont need to check it 🙂